dpsctl/dpsctl_GUI.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
```

Comment:
I think you may have done the same mistake I did the first time I raised a PR on GitHub. When a PR is opened from master, any new commits on master will go into that PR even if they are pushed _after_ the point in time when the PR was raised. If this version of ```dpsctl_GUI.py``` is the version referred to in your other PR, you can keep it here and I'll close the other PR. Oh, could you rename the tool to lower case?